lib/src/having_matcher.dart

```diff
@@ -55,10 +57,11 @@ class HavingMatcher<T> implements TypeMatcher<T> {
       .addAll('', ' and ', '', _functionMatchers);
 }
 
-class _FunctionMatcher<T> extends CustomMatcher {
+@visibleForTesting
+class FunctionMatcher<T> extends CustomMatcher {
```

Comment:
Why was this change made?

Reply:
To make sure it's not exported, i think...

Reply:
Ah, it was a private types in public APIs lint...